EIPS/eip-4393.md

```diff
@@ -310,6 +310,8 @@ deposits can remain where they are when a user tips, and only transferred out to
 
 This standard does not specify the type of ERC-20 compatible deposits allowed. Indeed, could be tip tokens themselves. But it is RECOMMENDED that balances of the deposits be checked after transfer to find out the exact amount deposited to keep internal accounting consistent. In case, for example, the ERC-20 contract takes fees and hence reduces the actual amount deposited.
 
+This standard does not specify any functionality for refunds for deposits nor for tip tokens sent, it is left to the implementer to add this to their smart contract(s). The reasoning for this is to keep the interface light and not to enforce upon implementers the need for refunds but to leave that as a choice.
+
```

Comment:
Changes to existing EIPs should be done in their own PR, not as part of a new EIP PR.
```suggestion
```